demo/index.html

```diff
@@ -90,21 +90,19 @@
     </style>
   </custom-style>
 
-  <dom-bind>
-    <template is="dom-bind" id="scope">
-      <dom-repeat>
-        <template is="dom-repeat" id="iconsetRepeat">
-          <h2>{{item.name}}</h2>
-          <dom-repeat class="set">
-            <template is="dom-repeat" items="{{getIconNames(item)}}">
-              <span class="container">
-                <iron-icon icon="{{item}}"></iron-icon>
-                <div>{{item}}</div>
-              </span>
-            </template>
-          </dom-repeat>
-        </template>
-      </dom-repeat>
+  <dom-bind id="scope">
+    <template is="dom-bind">
+      <template is="dom-repeat" id="iconsetRepeat">
+        <h2>{{item.name}}</h2>
+        <div class="set">
```

Comment:
Just double checking if you wanted to put the class `set` here on the div and not on the `dom-repeat` like it was before. I assume it's ok, because the dom-repeat doesn't need styling, but just double checking it wasn't a bad copy paste.

Reply:
Yup, this is meant to be a wrapper around the `dom-repeat`.